bin/build-helper

```diff
@@ -5,8 +5,11 @@ set -u
 set -o pipefail
 
 BASE_DIR=$(dirname $0)/..
-NODE_ENV=${NODE_ENV:-""}
-REGISTRY=${REGISTRY:=""}
+: ${NODE_ENV:=""}
+: ${REGISTRY:=""}
```

Comment:
`:` is a noop in shell.  These just set the globals with default empty values if not already assigned in the script environment.